CHANGELOG.md

```diff
@@ -4,44 +4,56 @@
 
 ### New features
 
-#### Create custom width container classes
+#### Set a specific width for wrappers
 
-You can now create custom page width container classes using the `govuk-width-container` mixin. You do this by passing in the required maximum width of the container.
+You can now:
+
+- create your own Sass classes with specific widths
+- add those classes to the `.govuk-width-container` wrapper using Nunjucks
+
+This means you can set specific widths for the:
+
+- [page wrapper](https://design-system.service.gov.uk/styles/layout/#page-wrappers)
+- [header component](https://design-system.service.gov.uk/components/header/)
+- [footer component](https://design-system.service.gov.uk/components/footer/)
+
+Include the `govuk-width-container` mixin inside your Sass class definition, and pass in the width in pixels.
```

Comment:
> Sass class definition

I've not heard 'Sass class definition' before, is this correct terminology for this code?

Could consider something simpler like

"Include the `govuk-width-container` mixin inside your code"

Reply:
Sounds good! Just for future reference - what would be the best term for one of these blocks in a Sass file?